package.json

```diff
@@ -1,4 +1,6 @@
 {
+  "name": "css-modules-flow-types",
```

Comment:
Not sure how this works with Lerna? Can you pull it out of this PR. Feel free to create a github issue on it. : )